client/src/components/landing/landing.css

```diff
@@ -98,6 +98,7 @@ figcaption.caption {
 /* testimonials */
 
 .testimonials-row {
+  --image-size: min(320px, 350px);
```

Comment:
Had to create another variable because the viewport is different, so I can't create one selector for all of them.